index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+
   <title>Volleyball</title>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="stylesheet" href="styles.css" />
   <script type="module" src="script.js"></script>
   <link rel="icon" href="/images/favicon.ico">
-  <style>
+
+  <!-- <style>

```

Comment:
I'd recommend just removing this commented-out code entirely if all your styles are now in styles.css